src/components/customerCases/customerCase/customerCase.module.css

```diff
@@ -8,33 +8,75 @@
 .content {
   display: flex;
   flex-direction: column;
-  max-width: 1200px;
```

Comment:
Why remove the max-width? Makes it pretty hard to read

<img width="1918" alt="image" src="https://github.com/user-attachments/assets/19682892-df73-44a3-b909-1cceb69161e6">